packages/obonode/obojobo-chunks-numeric-assessment/converter.js

```diff
@@ -0,0 +1,154 @@
+import Common from 'obojobo-document-engine/src/scripts/common'
+
+import {
+	SCORE_RULE_NODE,
+	NUMERIC_FEEDBACK_NODE,
+	EXACT_ANSWER,
+	WITHIN_A_RANGE,
+	PRECISE_RESPONSE,
+	MARGIN_OF_ERROR
+} from './constant'
+
+const complexifiedNumericRule = numericRule => {
```

Comment:
I'm not totally clear on the differences between complexifiedNumericRule and simplifiedNumericRule. NumericRule is an existing class with a defined set of properties but neither the simple or complex version here don't have the same properties as a NumericRule instance.